docs/guides/audio-config.md

```diff
@@ -20,33 +20,20 @@ If not present, you'll have to update your bootup kernel params:
 
 # Audio Configuration Files
 
-In most scenarios, you should use [these files](https://gist.github.com/MCMrARM/c357291e4e5c18894bea10665dcebffb), following the instructions in that gist's `README.md`.
+Simply run the following to set up audio :-
 
-Special scenarios are:
-
-- [2019 16" MacBook Pro audio files](https://gist.github.com/kevineinarsson/8e5e92664f97508277fefef1b8015fba) - that laptop has 6 speakers and needs slightly different config
-- [2020 13" MacBook Air audio files](https://gist.github.com/bigbadmonster17/8b670ae29e0b7be2b73887f3f37a057b)
+```bash
+git clone https://github.com/kekrby/t2-better-audio.git
+cd t2-better-audio
+bash ./install.sh
+sudo rm -r $HOME/t2-better-audio
+```
 
 # Using Pipewire instead of Pulseaudio
```

Comment:
can this section be removed? Or replaced with just a note on pulseaudio?

Reply:
We need this section cause Pulseaudio doesn't work with these?

Reply:
Oh it's still not working with pulseaudio. Maybe include a link to the previous version of this page for those who don't want to switch?

Reply:
Well, the previous version won't work as bce is also gonna get patched.

Reply:
Hmm they'd have to rename the files i guess. I'm not sure if we should break support for pa, would that be a regression?

Reply:
We could use kekrby's old config files then.

Reply:
[These files](https://github.com/kekrby/t2-better-audio/tree/163ecc6a0dbafbe8107fd5369bb374e343be9c61) should work with `pulseaudio` and an updated `apple-bce`.

Reply:
A `pulseaudio` with [this patch](https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/596.patch) applied should work as well.